index.html

```diff
@@ -1887,324 +1887,639 @@ <h2>
-          country as an upper case [[!ISO3166]] alpha-2 code, or to the empty
-          string if none was provided.
-          </li>
-          <li>Set the <var>address</var>.<a>[[\phone]]</a> to the user-provided
-          phone number for this shipping address, optionally formatted to
-          adhere to [[!E.164]], or to the empty string if none was provided.
-            <aside class="note" title="Privacy of phone number">
-              <p>
-                To maintain user's privacy, implementers need to be mindful
-                that a shipping address's associated phone number might be
-                different or the same from that of the end-user's. As such,
-                implementers need to take care to not provide the end user's
-                phone number to the merchant without the end user's consent.
-              </p>
-            </aside>
-          </li>
-          <li>Set <var>address</var>.<a>[[\languageCode]]</a> to a
+        <section>
+          <h2>
+            Constructor
+          </h2>
+          <p>
+            The steps to <dfn data-lt="PaymentAddress.PaymentAddress()"
+            data-no-default="">construct a <code>PymentAddress</code></dfn>
+            with <a>AddressInit</a> <var>details</var> are given by the
+            following algorithm:
+          </p>
+          <ol data-link-for="AddressInit">
+            <li>Let <var>address</var> be a new instance of
+            <a>PaymentAddress</a> with <a>[[\addressLine]]</a> set to an empty
+            <code><a data-cite=
+            "!WEBIDL#idl-sequence">sequence</a>&lt;<a data-cite=
+            "WEBIDL#idl-DOMString">DOMString</a>&gt;</code>, and all other
+            internal slots set to the empty string.
+            </li>
+            <li>If <var>details</var> was not passed to the constructor, return
+            <var>address</var> and terminate this algorithm.
+            </li>
+            <li>If <var>details</var>["<a>country</a>"] is present:
+              <ol>
+                <li>Let <var>country</var> be the result of <a data-cite=
+                "!INFRA#ascii-uppercase">ASCII uppercasing</a>
+                <var>details</var>["<a>country</a>"].
+                </li>
+                <li>If <var>country</var> is not a valid [[!ISO3166]] alpha-2
+                code, throw a <a>RangeError</a>.
+                </li>
+              </ol>
+            </li>
+            <li>If <var>details</var>["<a>addressLine</a>"] is present:
+              <ol>
+                <li>Push each item of <var>details</var>["<a>addressLine</a>"]
+                in order into <var>address</var>.<a>[[\addressLine]]</a>.
+                </li>
+              </ol>
+            </li>
+            <li>Freeze <var>details</var>["<a>addressLine</a>"].
+            </li>
+            <li>If <var>details</var>["<a>region</a>"] is present, set
+            <var>address</var>.<a>[[\region]]</a> to the result of trimming
```

Comment:
What is "trimming"? :) I think you want https://infra.spec.whatwg.org/#strip-leading-and-trailing-ascii-whitespace